HTML/layout1.css

```diff
@@ -0,0 +1,160 @@
+body
+{
+    margin: 0;
+    padding: 0;
+    background-color: #F2F2F2;
+}
+.navbar
+{
+    display: flex;
+    column-gap: 90%;
+    width: 100%;
+    height: 40px;
+    border: 1px;
+    background-color: white;
+}
+#logo
+{
+    margin-left: 5px;
+    width: 100px;
+    cursor: pointer;
+    height: 40px;
+}
+#search_1
+{
+    width: 100%;
+    height: 100%;
+}
+.search1
+{
+    padding: 0;
+    margin-right: 5px;
+    width: 50px;
+    height: 40px;
+    border: 0;
+    align-self: flex-end;
+    cursor: pointer;
+}
+.container1
+{
+    position: relative;
+    margin: 0 10% 0 10%;
+}
+#london
+{
+    width: 100%;
+    height: 100%;
+}
+.card
+{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    position: absolute;
+    width: 55%;
+    left: 22%;
```

Comment:
Restructure the css. use containers to bound the layouts

Reply:
What does this mean ???